finalPrep/modZeroHero.js

```diff
@@ -1,52 +1,77 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = "Buzz";
+var specialAbility = "Super Bee Powers";
 // Declare two variables - greeting AND catchphrase
+
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
-
+var greeting = (heroName + " has arrived!");
```

Comment:
Not necessary/really JS convention to wrap what is on the right side of the assignment operator in `()`, but it works and is OK. Just giving you a heads up because you'll want a consistent coding style throughout a code file. When working on teams, you'll need to norm on those things!